lib/web/terminal.go

```diff
@@ -788,6 +797,10 @@ func (t *TerminalHandler) streamEvents(ws *websocket.Conn, tc *client.TeleportCl
 			err = ws.WriteMessage(websocket.BinaryMessage, envelopeBytes)
 			t.wsLock.Unlock()
 			if err != nil {
+				if errors.Is(err, websocket.ErrCloseSent) {
+					logger.WithError(err).Debug("Websocket was closed, no longer streaming events")
+					return
```

Comment:
I think this needs to be `continue`

Reply:
Why? The loop is writing to the websocket. If the websocket is closed and you continue to loop you will continue to get "websocket already closed" errors, right?

Reply:
Sorry this was just a comment for the test debugging, you're right that this should be `return`.